tools/logger/convert.c

```diff
@@ -141,6 +141,13 @@ get_uuid_entry(const struct snd_sof_uids_header *uids_dict, uint32_t uid_ptr)
 	       uids_dict->base_address);
 }
 
+uint32_t get_uuid_key(const struct snd_sof_uids_header *uids_dict,
```

Comment:
static inline ?

Reply:
This function is strictly related with `get_uuid_entry`, they just work in opposite directions, so I would like to keep them together, but only one of them is used in other file, so I will keep it as it is.